.github/workflows/CodeQuality.yaml

```
@@ -8,40 +8,57 @@ jobs:
-      run: docker compose run --rm app npm run e2e-test
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Copy .env file
+        run: cp .env.example .env
+
+      - name: Docker starts services
+        run: docker compose up -d --build
+
+      - name: Node version check
+        run: docker exec -it $(docker ps -l -q) node -v
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:17: Quote this to prevent word splitting

(shellcheck)